twitter/twitter.php

```diff
@@ -1919,7 +1919,7 @@ function twitter_fetchhometimeline(App $a, $uid)
 
 			$notify = false;
 
-			if (($postarray['uri'] == $postarray['thr-parent']) && ($postarray['author-link'] == $postarray['owner-link'])) {
```

Comment:
I can imagine that this "author-link-owner-link" stuff does have a reason. Why had you removed it?

Reply:
In the `twitter_createpost` function, we initialize `author-link` to be the same as `owner-link`, so this condition is always true. I can check how long we've been doing this but it definitely predates my last changes with `thr-parent`.

Reply:
I had a look at the code. I'm mostly sure that it is some left over from the time reshares looked different.